templates/app/aspnet-core/src/MyCompanyName.MyProjectName.Blazor.Server/MyProjectNameBlazorModule.cs

```diff
@@ -206,9 +206,9 @@ private void ConfigureSwaggerServices(IServiceCollection services)
 
         private static void ConfigureHttpClient(ServiceConfigurationContext context)
         {
-            context.Services.AddTransient(sp => new HttpClient
+            context.Services.AddHttpClient("abp", sp => 
```

Comment:
You can consider use constant string.

Reply:
Sure, you mean something defined in MyProjectNameConsts like DbTablePrefix or inside Blazor/Blazor.Server, since is required only in this UI version?

Reply:
Can you share the document that explains the `AddHttpClient`?

Reply:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-5.0#httpclient-and-lifetime-management

Reply:
> When I tried to use the Syncfusion SfDataManager control, I have found that a little change is required to solve URI exception, 

Hi 

Does Syncfusion have a document explain it?

Can you share the exception details?

Reply:
I don't have any Syncfusion document for this problem, I found the solution myself by following Microsoft's recommendations for httpclient.

I attached sample to reproduce the following exception "System.UriFormatException: 'Invalid URI: The format of the URI could not be determined.'"
[SfDataManager.Server.zip](https://github.com/abpframework/abp/files/7228103/SfDataManager.Server.zip)